backend/src/main/java/gov/cdc/usds/simplereport/api/model/filerow/PatientUploadRow.java

```diff
@@ -45,35 +45,73 @@ public class PatientUploadRow implements FileRow {
   final ValueOrError role;
   final ValueOrError email;
 
+  static final String firstNameField = "first_name";
```

Comment:
⚠️ sonarcloud flagged this as a code smell previously since it didn't want me repeating the string 3 times so I made it into a variable